internal/request/graphql/schema/types/base.go

```diff
@@ -40,6 +40,28 @@ func BooleanOperatorBlock() *gql.InputObject {
 	})
 }
 
+// BooleanListOperatorBlock filter block for [Boolean] types.
+func BooleanListOperatorBlock(op *gql.InputObject) *gql.InputObject {
+	return gql.NewInputObject(gql.InputObjectConfig{
+		Name:        "BooleanListOperatorBlock",
+		Description: "These are the set of filter operators available for use when filtering on [Boolean] values.",
+		Fields: gql.InputObjectConfigFieldMap{
+			"_any": &gql.InputObjectFieldConfig{
+				Description: anyOperatorDescription,
+				Type:        op,
+			},
+			"_all": &gql.InputObjectFieldConfig{
+				Description: allOperatorDescription,
+				Type:        op,
+			},
+			"_none": &gql.InputObjectFieldConfig{
+				Description: noneOperatorDescription,
+				Type:        op,
+			},
+		},
```

Comment:
suggestion: this can be extracted into a function and used everywhere

Reply:
I have mixed feelings about the extraction of the full `InputObjectFieldConfig` blocks to a shared func (it couples them together, and hides what each InputObject contains, yet the enforcement of consistency is nice).

Reply:
My preference is to keep them as is because it makes modifying each individual type easier.